<!--
ROB599 Homwork 1 - Staring at walls
Jake Anderson
Oct 24, 2020
-->


<launch>

	<!-- Start the laser filter node. -->
	<node pkg="rob599_hw1" name="laser_filter" type="laserScan_filter.py" >
	</node>

	<!-- Start the RVIZ marker node. -->
	<node pkg="rob599_hw1" name="lidar_marker" type="markers.py" >
	</node>

	<!-- Start the wall approach node. -->
	<node pkg="rob599_hw1" name="staring_at_wall" type="wall_starer.py">
	</node>

	<!-- Start the wall finder node. -->
	<node pkg="rob599_hw1" name="wall_finder" type="find_wall.py">
	</node>

</launch>
